### ISSUE TITLE
Generated Cucumber JSON missing the uri property

### ISSUE THREAD

**Issue by rdasan** [picklesdoc/pickles#609]
Hi I'm using Pickles to generate `Cucumber Json` format from my Test Results file. However I see that the generated `cucumberResults.json` is missing the `uri` property. I am planning to use the generated cucumber json to publish the cucumber results back into a BDD JIRA plugin and that plugin requires the correct Cucumber JSON format that includes a `uri` property in the generated Cucumber Json.

I see that this issue has been mentioned earlier in this closed issue : https://github.com/picklesdoc/pickles/issues/278

https://github.com/picklesdoc/pickles/issues/278#issuecomment-317451315

Are there any plans to support this moving forward?

**Comment by isitgeorge** [comment]
+1

Also missing `id` properties from both the top level feature and child objects of the `elements` array.

**Comment by ericcallsynx** [comment]
Yes, I would like to address this issue. Most of the issues that Dirk closed before he handed off the project will be reopened and worked eventually. Most of my time so far has been spent trying to get the dotnet standard/core version working. I have most of it functional but still have some unit tests failing. Once I get past that roadblock I'll start addressing more of the functionality issues.

**Comment by andrey-covergo** [comment]
I have the same issue as @rdasan as now the [only way](https://github.com/hindsightsoftware/hotel-booking-specflow-dotnet-example) to [convert SpecFlow results](https://help.hindsightsoftware.com/en/articles/4771876-test-reports-with-specflow-and-net) into Cucumber json is via official [SpecFlow runner SpecRun](https://specflow.org/tools/runner/), requiring online account binding and relying on Mono, also conflicting with SpecFlow xUnit runner. My use case is Linux-based CI and MacOS-based development environment, so I look into Pickles or [SpecNuts](https://github.com/DonutReport/SpecNuts).
